File: go-dtm-server/models/task.go
package models

type Task struct {
	Id           int	       `json:"id"`
	Creator      *User         `json:"creator,omitempty"`
	Asignee      *User         `json:"asignee,omitempty"`
	AsigneeId    *int          `json:"asigneeId,omitempty"`
	TaskType     *Type         `json:"type"`
	TaskTypeId   *int          `json:"typeId,omitempty"`
	TaskStatus   *Type         `json:"status"`
	TaskStatusId *int          `json:"statusId,omitempty"`
	Title        string        `json:"title"`
	Text         string        `json:"text,omitempty"`
	CreationDate int32         `json:"creationDate"`
	StartDate    int32         `json:"startDate"`
	EndDate      int32         `json:"endDate"`
	UpdateDate   int32         `json:"updDateTime"`
	Attachments  []*Attachment `json:"attachments"`
	Comments     []*Comment    `json:"comments"`
}

type Tasks struct {
	Tasks []*Task `json:"tasks"`
}

File: go-dtm-client/GoDtmUi/Models/Task.cs
using Newtonsoft.Json;



namespace GoDtmUI.Models
{
    public class Task
    {
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("creator")]
        public User Creator { get; set; }

        [JsonProperty("asignee")]
        public User Asignee { get; set; }

        [JsonProperty("type")]
        public Status Type { get; set; }

        [JsonProperty("status")]
        public Status Status { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; }

        [JsonProperty("text")]
        public string Text { get; set; }

        [JsonProperty("creationDate")]
        public int CreationDate { get; set; }

        [JsonProperty("startDate")]
        public int StartDate { get; set; }

        [JsonProperty("endDate")]
        public int EndDate { get; set; }

        [JsonProperty("updDateTime")]
        public int UpdDateTime { get; set; }

        [JsonProperty("attachments")]
        public Attachment[] Attachments { get; set; }

        [JsonProperty("comments")]
        public Comment[] Comments { get; set; }
        
        
        public static Task FromJson(string json) => JsonConvert.DeserializeObject<Task>(json, Converter.Settings);
    }
}